src/Core/Localization/Currency/CurrencyDataSource.php

```diff
@@ -62,4 +72,35 @@ public function getDataByCurrencyCode($currencyCode)
     {
         return $this->topLayer->read($currencyCode);
     }
+
+    /**
+     * Is this currency available ?
+     * (an available currency is not deleted AND is active)
+     *
+     * @param $currencyCode
+     *
+     * @return bool
+     *  True if currency is available
```

Comment:
why do you put your comment in another line? It's not invalid but something like the following is more common:

```
* @return bool True if the currency is available.
```

What do you think?